2024/political-donations-by-party/main.js

```diff
@@ -6,8 +6,8 @@ var drawGraph = function () {
     var svgBackgroundColor = '#ffffff';
 
     //width and height of the SVG
-    const width = 900;
-    const height = 500;
+    const width = 650;
+    const height = 400;
```

Comment:
Were you able to validate that this would work?

Reply:
no, how would you recommend I do this?

Reply:
Ping me on Slack if that doesn't make sense and I can walk you through it.

Reply:
For this to work, your dimensions are gonna need to be something like this:
```css
width: 100%;
height: 400px;
```

If the width isn't dynamic, it will choke off at some point and give you something like this:
<img width="561" alt="Screenshot 2024-02-07 at 10 30 12 PM" src="https://github.com/chicagomaroon/data-visualizations/assets/5885605/60a52c5d-d000-465e-8c83-61fea46a3423">